README.md

```diff
@@ -92,9 +92,11 @@ moment.loadPersian([options])
 
 ### Options
 
-| Param            | Type    | Default | Description                               | Example                                      |
-| ---------------- | ------- | ------- | ----------------------------------------  | -------------------------------------------- |
+| Param            | Type    | Default   | Description                               | Example                                      |
+| ---------------- | ------- | -------   | ----------------------------------------  | -------------------------------------------- |
 | usePersianDigits | Boolean | `false`   | Use persian digits (Use at your own risk) | `moment.loadPersian({usePersianDigits: true})` |
+| ---------------- | ------- | -------   | ----------------------------------------  | -------------------------------------------- |
```

Comment:
Remove this line. it's not needed between the rows. Only after the header. ( view the file to see the bug)